app/assets/javascripts/game.js

```diff
@@ -54,12 +58,22 @@ function create() {
 
     game.add.tween(logo).to( { alpha: 1 }, 2000, Phaser.Easing.Linear.None, true);
     debugText = game.add.text(game.world.centerX, game.world.centerY, "",{ font: "12px Arial", fill: "#ff0044", align: "left" });
+    testButtonText = game.add.text(game.world.centerX, game.world.centerY, "",{ font: "12px Arial", fill: "#ff0044", align: "left" });
     diePoolText = game.add.text(game.world.centerX, game.world.centerY, "",{ font: "12px Arial", fill: "#ff0044", align: "center", wordWrap: true, wordWrapWidth: 100 });
     playerText = game.add.text(game.world.centerX, game.world.centerY, "",{ font: "12px Arial", fill: "#ff0044", align: "center", wordWrap: true, wordWrapWidth: 100 });
 
     debugText.fixedToCamera = true;
     debugText.cameraOffset.setTo(200, 500);
 
+    testButtonText.fixedToCamera = true;
+    testButtonText.cameraOffset.setTo(10, 50);
+
+    // shows the die group
+    dieGroup = game.add.group();
+    dieSpriteGroup = new SpriteGroup("dice", dieGroup, diePool, 6, 900, 700);
```

Comment:
creates a SpriteGroup object, can be used to render a group of sprites. See 50ac87b669958ac422aed996413e7b5a0ba88865.